.github/workflows/build.yml

```
@@ -26,6 +26,7 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
+
       - name: Cache west modules
         uses: actions/cache@v3
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)